pictures/tasks.py

```diff
@@ -112,3 +112,30 @@ def process_picture(field_file: PictureFieldFile) -> None:  # noqa: F811
                 queue=conf.get_settings().QUEUE_NAME,
             )
         )
+
+
+try:
+    from django_rq import job
+except ImportError:
+    pass
+else:
+
+    @job
```

Comment:
I think you need to specify a queue here. More infos down below.

Reply:
I defined queue here